custom/icds_reports/sqldata.py

```diff
@@ -893,8 +948,12 @@ def columns(self):
                 'Weighing efficiency',
                 percent,
                 [
-                    SumColumn('nutrition_status_weighed'),
-                    SumColumn('wer_eligible')
+                    SumColumn('nutrition_status_weighed', filters=self.filters + [
+                        RawFilter("age_tranche != '72'")
```

Comment:
Why are you using multiple formats for the same filters? Is there ar eason they can't all be of the format:

```
NOT(EQ('age_tranche', '72'))
```